config/migrations/20250212061535-insert-holiday-concepts.ttl

```diff
@@ -0,0 +1,42 @@
+@prefix mu: <http://mu.semte.ch/vocabularies/core/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+<http://timekeeper.redpencil.io/concept-schemes/8d0407eb-36b9-4ef5-b6df-e63df460446d> a skos:ConceptScheme ;
+    mu:uuid "8d0407eb-36b9-4ef5-b6df-e63df460446d" ;
+    skos:prefLabel "Holiday Counters" .
+
+<http://timekeeper.redpencil.io/concepts/6644c724-96b8-446f-8aec-ced33577c639> a skos:Concept ;
+    mu:uuid "6644c724-96b8-446f-8aec-ced33577c639" ;
+    skos:prefLabel "Holiday Counter" ;
+    skos:inScheme <http://timekeeper.redpencil.io/concept-schemes/8d0407eb-36b9-4ef5-b6df-e63df460446d> ;
+    skos:topConceptOf <http://timekeeper.redpencil.io/concept-schemes/8d0407eb-36b9-4ef5-b6df-e63df460446d> .
+
+<http://timekeeper.redpencil.io/concepts/eb9813ca-cfe5-444b-94b9-09874982d92c> a skos:Concept ;
+    mu:uuid "eb9813ca-cfe5-444b-94b9-09874982d92c" ;
+    skos:prefLabel "Public holiday in weekend" ;
+    skos:inScheme <http://timekeeper.redpencil.io/concept-schemes/8d0407eb-36b9-4ef5-b6df-e63df460446d> ;
+    skos:broader <http://timekeeper.redpencil.io/concepts/6644c724-96b8-446f-8aec-ced33577c639> .
```

Comment:
Conflicts with the `skos:broader` predicate that is used in the mu-cl-resource configuration. Doesn't matter which one you use, but should be consistent. Maybe check if there are already `skos:broader`/`skos:narrower` predicates in the triplestore and use the same.